zcash_transparent/src/builder.rs

```diff
@@ -28,6 +30,7 @@ pub enum Error {
     InvalidAmount,
     /// A bundle could not be built because a required signing keys was missing.
     MissingSigningKey,
+    InvalidOpReturn,
```

Comment:
- Document this new variant.
- Document this change to the enum in the changelog (adding variants to a public enum is a breaking change if it doesn't have a `nonexhaustive` annotation).